syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py

```diff
@@ -206,11 +207,18 @@ def map_key(k: str) -> str:
         else:
             return config
 
-    def _active_hyperparameters(self, configuration: Dict[str, Any]) -> List[str]:
+    def _get_active_hyperparameters(self, configuration: Dict[str, Any]) -> List[str]:
+        # Some of the hyperparameters are only active for certain values of other hyperparameters
+
+        config_with_fidelity = {
```

Comment:
This warrants a comment, as it is pretty odd that fidelity is set to some value here. I see now what this is doing, but it is not obvious.